packages/backend/src/account/service.ts

```diff
@@ -11,26 +11,17 @@ import {
 import { Pagination } from '../shared/pagination'
 import { TransferService, TwoPhaseTransfer } from '../transfer/service'
 import { TransferError, TransfersError } from '../transfer/errors'
-import { AccountError, AccountTransferError, UnknownAssetError } from './errors'
+import { AccountTransferError, UnknownAssetError } from './errors'
 import { Account } from './model'
 
 export { Account }
 
-export type Options = {
+export interface CreateOptions {
   disabled?: boolean
-}
-
-type CreateAccountOptions = Options & {
   assetId: string
   sentBalance?: boolean
 }
 
```

Comment:
I'm wondering if the enabling/disabling of an account is a common enough action that it may make sense to just make it a method on the service? ie `enabled` and `disable` that take in the accountId. The bulk patch method is nice, but also not as explicit when being used to enable and disable an account.

Reply:
I went ahead and removed the bulk patch since the only remaining update-able field was `disabled`.
I imagine payment and peer services can either use an `accountService.disable`/`enable` or patch the `account.disabled` directly.
For example, I'm not sure if outgoing payments already have the account available for update?
https://github.com/interledger/rafiki/blob/main/packages/backend/src/outgoing_payment/worker.ts#L51-L81